libyaml/src/Text/Libyaml.hs

```diff
@@ -668,13 +675,17 @@ parserParseOne' parser = allocaBytes eventSize $ \er -> do
 -- @since 0.10.2.0
 data FormatOptions = FormatOptions
     { formatOptionsWidth :: Maybe Int
+    , formatOptionsExplicitMappingTags :: Bool
+    , formatOptionsExplicitSequenceTags :: Bool
```

Comment:
Looks good. My only question: would it be more flexible to instead have something like:

```haskell
data TagType = Explicit | Implicit

formatOptionsTagType :: Event -> TagType
```

Reply:
👍 On it.

I finally like this approach.

There might be even be a sane and safe unification of scalar and collection handling in reach...